docs/http.md

```diff
@@ -253,14 +254,23 @@ sources=0;5;7&destinations=5;1;4;2;3;6
 #### Example Request
 
 ```curl
-# Returns a 3x3 matrix:
+# Returns a 3x3 duration matrix:
 curl 'http://router.project-osrm.org/table/v1/driving/13.388860,52.517037;13.397634,52.529407;13.428555,52.523219'
 
-# Returns a 1x3 matrix
+# Returns a 1x3 duration matrix
 curl 'http://router.project-osrm.org/table/v1/driving/13.388860,52.517037;13.397634,52.529407;13.428555,52.523219?sources=0'
 
-# Returns a asymmetric 3x2 matrix with from the polyline encoded locations `qikdcB}~dpXkkHz`:
+# Returns a asymmetric 3x2 duration matrix with from the polyline encoded locations `qikdcB}~dpXkkHz`:
 curl 'http://router.project-osrm.org/table/v1/driving/polyline(egs_Iq_aqAppHzbHulFzeMe`EuvKpnCglA)?sources=0;1;3&destinations=2;4'
+
+# Returns a 3x3 duration matrix:
+curl 'http://router.project-osrm.org/table/v1/driving/13.388860,52.517037;13.397634,52.529407;13.428555,52.523219&annotations=duration'
+
+# Returns a 3x3 duration matrix and a 3x3 distance matrix:
+curl 'http://router.project-osrm.org/table/v1/driving/13.388860,52.517037;13.397634,52.529407;13.428555,52.523219&annotations=distance'
+
```

Comment:
Ok wait. no. This is confusing. so

`annotation=distance` and `annotation=duration,distance` will  return the same response?

If yes, and this is intended we should make it clearer. this is confusing.
If yes, and this is not 100% thought through can we discuss?

Reply:
@TheMarex actually maybe you can clarify this. In the comment here: https://github.com/Project-OSRM/osrm-backend/pull/4990#issuecomment-380067808 I took this to mean that we should always return `duration` table whether it was asked for or not in the response. That's what I've implemented and what I've documented here.